subsys/compress/include/nrf_compress/implementation.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
```

Comment:
The header documentation is not following the doxygen guidelines: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/dev_model_and_contributions/documentation/styleguide.html#doxygen_guidelines

Please include the file headers and groups and the correctly formatted documentation for different elements.

Example of a well documented header: https://github.com/nrfconnect/sdk-nrf/blob/main/include/dfu/dfu_multi_image.h

Reply:
Will address that in a revised commit once API design is agreed upon